### ISSUE TITLE
Add alias to helper class_variants method

### ISSUE THREAD

**Issue by javierav** [avo-hq/class_variants#27]
Now the `ClassVariants::Helper` helper uses the class and instance method name `class_variants` to define and invoke the variants, but when that is used in a class, for example a component, it might be more descriptive to call it `style`. What do you think @adrianthedev ?

```ruby
class MyClass
  include ClassVariants::Helper

  class_variants do
    ...
  end
end

MyClass.new.class_variants(:container, color: :red, class: "shadow")
```

vs

```ruby
class MyClass
  include ClassVariants::Helper

  style do
    ...
  end
end

MyClass.new.style(:container, color: :red, class: "shadow")
```

**Comment by adrianthedev** [comment]
I would keep away from `style` as it might be used to add CSS styles (who knows what API the user comes up with).

My question is, does it clash with anything? Do we need to change it?

**Comment by javierav** [comment]
No, there is no need to change anything! It's just that perhaps using the `class_variants` keyword to define and get the styles, I don't know if it could be improved with another more suitable keyword or not.
